### ISSUE TITLE
Azerbaijani ISO code

### ISSUE THREAD

**Issue by drammock** [phoible/phoible#67]
The "gold standard" document lists language `azb` "Azerbaijani" (SPA).  The current output of the aggregation script has the same data (from SPA) under ISO code `azj`.  Which is correct?  Did you change this in the gold standard doc or in the SPA raw data file at some point, @bambooforest?  I couldn't find any open or closed issues mentioning those ISO codes or language name.

Additional info: according to [the ISO table](http://www-01.sil.org/iso639-3/codes.asp?order=639_3&letter=A), `azb` is "North Azerbaijani" and `azj` is "Sourth Azerbaijani", and there is also a macrolanguage `aze`.

**Comment by bambooforest** [comment]
If look at what WALS tagged this resource:

http://wals.info/refdb/record/Householder-and-Lofti-1965

(check the bibtex entry) we get both azb and azj. I think at some point we were avoiding macrocodes, but in this case it seems like the best choice. Thoughts?

**Comment by drammock** [comment]
It seems like in this case the macrocode might be the appropriate choice.
However, if we do that I think we need to build an additional column into
the aggregation script that says "this is a macrolanguage" so that users
can easily exclude macrolanguages if they want to (or look only at the
macrolanguages).  We may want to do something similar to make it easy to
find all signed / mixed / pidgin / extinct languages too...  I know we had
a way to exclude those when we wrote the _Language_ paper, so it should be
fairly easy to dig up those lists of ISO codes.  Leave this issue open and
I'll resolve it in a couple weeks.

**Comment by drammock** [comment]
I went to the source, which clearly says in the introduction that

> The dialect presented in this book is as closely Tabrizi as possible...

Where Tabrizi is the dialect of Tabriz, i.e., Southern Azerbaijani (cf. the first few paragraphs of [this review article](http://www.jstor.org/stable/597735?seq=1#page_scan_tab_contents)).  Apparently the book also has one chapter on the dialect of Baku (which would be Northern Azerbaijani), but I looked at it and it's too cursory to have been the source of the phoneme data.  So I think we should code this as `azj`
